portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java

```diff
@@ -181,8 +181,12 @@ else if (PATH_PORTLET_FILE_ENTRY.equals(pathArray[0])) {
 				}
 			}
 			else {
-				long groupId = _getGroupId(user.getCompanyId(), pathArray[0]);
+				Group group = _getGroup(user.getCompanyId(), pathArray[0]);
+
+				_checkDirectoryIndexingEnabled(group);
```

Comment:
Hey @jcampoy I don't fully understand why we need this new check here and we don't need it in the others clauses of the conditional (previous else and else ifs)? However, I don't fully understand the semantic/purpose of Directory Indexing property. It seems it was included in https://issues.liferay.com/browse/LPS-22600. 
Maybe @csierra has more information/knowledge whether this is right/required or not?